CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # CHANGELOG
 
 ## Unreleased
-
+- [#108](https://github.com/georust/gpx/pull/108): Update MSRV to 1.74.
```

Comment:
This doesn't actually bump `rust-version` in the manifest, though I suspect it still builds on older versions?

Reply:
> This doesn't actually bump `rust-version` in the manifest, though I suspect it still builds on older versions?

I updated the changelog in f23ed65. This PR only changes the CI toolchain.

Reply:
"Yes we support older versions, but we don't know how to build them" doesn't make sense to me.

I'd say bump the rust-version, call it an MSRV and move on with your lives.

The alternative you're leading us down here is to have an MSRV that doesn't mean anything to anyone. How would we know when it breaks?

Reply:
To clarify: Whatever is listed as `rust-version` in Cargo.toml should have a corresponding building in CI. If that means you need to bump rust-version, (I think) that's OK.

Reply:
But if we switch to rustup, we might not need to bump the MSRV.